schemas/json/component/common-defs.schema.v1.json

```diff
@@ -0,0 +1,607 @@
+        }
+      },
+      "required": [
+        "cells"
+      ]
+    },
+    "gridCellText": {
+      "properties": {
+        "text": {
+          "title": "Text",
+          "description": "The text or text resource ID to be rendered in the cell",
+          "type": "string"
+        }
+      },
+      "$ref": "#/$defs/tableColumnOptions",
+      "required": [
+        "text"
+      ]
+    },
+    "gridCellComponent": {
+      "properties": {
+        "component": {
+          "title": "Component ID",
+          "description": "The ID of the component to be rendered in the cell",
+          "type": "string"
+        }
+      },
+      "$ref": "#/$defs/tableColumnOptions",
+      "required": [
+        "component"
+      ]
+    },
+    "groupEditOptions": {
+      "properties": {
+        "mode": {
+          "title": "Edit mode",
+          "description": "Mode for how repeating group table is displayed in edit mode",
+          "type": "string",
+          "enum": [
+            "hideTable",
+            "likert",
+            "showAll",
+            "showTable",
+            "onlyTable"
+          ]
+        },
+        "filter": {
+          "title": "Filter",
+          "description": "Conditions for filtering visible items in repeating group",
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/groupFilterItem"
+          }
+        },
+        "saveButton": {
+          "title": "Save button",
+          "description": "Boolean or expression indicating whether save button should be shown or not for a given row",
+          "$ref": "expression.schema.v1.json#/definitions/boolean"
+        },
+        "saveAndNextButton": {
+          "title": "Save and open next button",
+          "description": "Boolean or expression indicating whether save and go to next button should be shown or not in addition to save and close button",
+          "$ref": "expression.schema.v1.json#/definitions/boolean"
+        },
+        "deleteButton": {
+          "title": "Delete button",
+          "description": "Boolean or expression indicating whether delete button should be shown or not for a given row",
+          "$ref": "expression.schema.v1.json#/definitions/boolean"
+        },
+        "editButton": {
+          "title": "Edit button",
+          "description": "Boolean or expression indicating whether edit button should be shown or not for a given row",
+          "$ref": "expression.schema.v1.json#/definitions/boolean"
+        },
+        "multiPage": {
+          "title": "Multi-page",
+          "description": "Boolean value indicating if form components in edit mode should be shown over multiple pages/views.",
+          "type": "boolean"
+        },
+        "addButton": {
+          "title": "Add button",
+          "description": "Boolean or expression indicating whether add new button should be shown or not under the table.",
+          "$ref": "expression.schema.v1.json#/definitions/boolean"
+        },
```

Comment:
You forgot to add `../layout/` in front of the reference to `expression.schema.v1.json#/definitions/boolean` here, so it cannot find the reference.
<img width="833" alt="image" src="https://github.com/Altinn/app-frontend-react/assets/47412359/bc898125-0ee3-4aea-bc07-8d1832730462">

Reply:
Whoops 🙈 Updated now.